src/Goerli-DssSpell.t.sol

```diff
@@ -83,6 +84,16 @@ contract DssSpellTest is GoerliDssSpellTestBase {
         assertEq(reg.symbol("WBTC-B"), "WBTC");
     }
 
+    function testAAVELerpOffboardings() public {
+        checkIlkLerpOffboarding("AAVE-A", "AAVE Offboarding", 165, 500, 600, 1000);
+    }
+    function testBALLerpOffboardings() public {
+        checkIlkLerpOffboarding("BAL-A", "BAL Offboarding", 165, 800, 1000, 1600);
+    }
+    function testCOMPLerpOffboardings() public {
+        checkIlkLerpOffboarding("COMP-A", "COMP Offboarding", 165, 250, 350, 500);
+    }
```

Comment:
I'd suggest making just one test for these ones. Taking into account they are the same subject makes sense to group them and will make the run faster.